joblib/test/test_parallel.py

```diff
@@ -27,33 +30,16 @@
 from joblib.test.common import with_multiprocessing
 from joblib.testing import (parametrize, raises, check_subprocess_call,
                             skipif, SkipTest, warns)
-from joblib._compat import PY3_OR_LATER, PY27
 
-try:
-    import cPickle as pickle
-except ImportError:
-    import pickle
 
-try:
-    from queue import Queue
-except ImportError:
-    # Backward compat
-    from Queue import Queue
+from queue import Queue
 
 try:
     import posix
 except ImportError:
     posix = None
 
-try:
-    RecursionError
-except NameError:
-    RecursionError = RuntimeError
-
-try:
-    reload         # Python 2
-except NameError:  # Python 3
-    from importlib import reload
+RecursionError
```

Comment:
We do not need this line. It was just used to see if we would trigger a `NameError` under Python 2.